src/stores/toast.store.ts

```diff
@@ -0,0 +1,24 @@
+import { defineStore } from "pinia";
+import { toast } from 'vue3-toastify';
+import 'vue3-toastify/dist/index.css';
+
+export const useToastStore = defineStore({
```

Comment:
A composable would be better than a store, but sure.
The id of the store isn't okay tho.